src/lib/tabs/tab-body.ts

```diff
@@ -53,6 +58,53 @@ export type MdTabBodyPositionState =
 export type MdTabBodyOriginState = 'left' | 'right';
 
 /**
+ * The portal host directive for the contents of the tab.
+ * @docs-private
+ */
+@Directive({
+  selector: '[matTabBodyHost]'
+})
+export class MdTabBodyPortal extends _MdTabBodyPortalBaseClass implements OnInit {
+  /** Whether the tab body should be visible. */
+  visible = false;
+  /** A subscription to events for when the tab body begins centering. */
```

Comment:
Looks like the subs should be private (preceeded with an underscore)

Reply:
Done